astroquery/splatalogue/tests/test_splatalogue.py

```diff
@@ -104,10 +104,12 @@ def test_band_crashorno():
 
 
 # regression test : version selection should work
+# Unfortunately, it looks like version1 = version2 on the web page now, so this
+# may no longer be a valid test
 @remote_data
 def test_version_selection():
     results = splatalogue.Splatalogue.query_lines(
         min_frequency=703 * u.GHz, max_frequency=706 * u.GHz,
         chemical_name='Acetaldehyde', version='v1.0')
 
-    assert len(results) == 1
+    assert len(results) == 133
```

Comment:
I really don't understand this.  It is entirely possible that a huge # of additional lines were added to the database.  However, since this is v1.0, I expected it to stay static.  I suspect they made some upstream changes that are counterintuitive.